src/main/webapp/partials/instructorHelpGettingStarted.jsp

```diff
@@ -331,8 +334,16 @@
             <li>Team peer evaluation session</li>
             <div style="margin: 0 auto; padding: 0 50px;">
               <ul>
-                <li>Provides 5 standard questions for team peer evaluations</li>
-                <li> Allows you to modify/remove the given questions and add your own questions as required</li>
+                <li>Provides 5 questions for team peer evaluations, with the first question being a contribution question</li>
+                <li>Allows you to modify/remove the given questions and add your own questions as required</li>
+              </ul>
+            </div>
+            <li>session using template: team peer evaluation (optimized)</li>
+            <div style="margin: 0 auto; padding: 0 50px;">
+              <ul>
+                <li>Provides 5 standard questions for team peer evaluations, with the first question being a distribute points question</li>
+                <li>Allows you to modify/remove the given questions and add your own questions as required</li>
+                <li>Recommended template for new users (distribute points questions are generally easier to use)</li>
```

Comment:
easier to use than contribution questions u mean...
I think you can also give a link to the question type help for distribute point question and contribution question here.

Reply:
@wenmogu Do you mean something like the following:

![image](https://user-images.githubusercontent.com/24493463/37751441-85c2990c-2dcd-11e8-8111-9a9366d92b31.png)

and

![image](https://user-images.githubusercontent.com/24493463/37751445-8c02d9e4-2dcd-11e8-96ce-c73b4c6ce860.png)

Reply:
Yuppp